### ISSUE TITLE
Move Platform Ant Remote jars to JavaSE 17

### ISSUE THREAD

**Issue by jukzi** [eclipse-platform/eclipse.platform#411]
There are still some files left referring JavaSE-11. See https://github.com/eclipse-platform/eclipse.platform/search?q=JavaSE-11

It feels like an analog to "Bug 578533 - Move Platform Ant Remote jars to JavaSE 11" https://bugs.eclipse.org/bugs/show_bug.cgi?id=578533 is missing for 17. (see commit https://github.com/eclipse-platform/eclipse.platform/commit/3df068b0ebf581c2b4ee6e9b2b939fa43532c4c5)

@SarikaSinha how to rebuild those jars?

**Comment by iloveeclipse** [comment]
@jukzi : if we move remote ant support to Java 17, we would not be able to run ant tasks on Java 11 anymore. This is not desired, if someone project is still uses on Java 11 and can't be executed on 17 (e.g. because of missing nashorn engine).

**Comment by merks** [comment]
Indeed, I saw on the forum just last week someone using an ancient version of Eclipse so that they can launch Ant with Java 1.6!
